ucuapi/protocols/dummy/module.py

```diff
@@ -8,12 +8,15 @@
     config = json.load(json_file)['config'][0]
 
 class Device:
-  def __init__(self, udid, values):
+  def __init__(self, udid, devType, values):
     self.udid = udid
+    self.devType = devType
     self.values = values
 
   def setValue(self, name, value):
     self.values[name] = value
+  def setType(self, name, devType):
```

Comment:
I don't know if we'll want this method. I don't think we'll be changing a device's type on-the-fly.

Reply:
I figured it would be used when devices are initialized, such as a new device is found + added after a scan. The device type would need to be set somehow.

Reply:
With the way we were looking to have everything set up, I believe devices would only be added via a scan. While it's not really affecting anything, though, I won't hold up the review on just a simple function removal.